routers/api/v1/api.go

```diff
@@ -1002,6 +1002,7 @@ func Routes(ctx gocontext.Context) *web.Route {
 						m.Get(".{diffType:diff|patch}", repo.DownloadPullDiffOrPatch)
 						m.Post("/update", reqToken(), repo.UpdatePullRequest)
 						m.Get("/commits", repo.GetPullRequestCommits)
+						m.Get("/files", context.ReferencesGitRepo(false), repo.GetPullRequestFiles)
```

Comment:
Does the `ReferencesGitRepo(allowEmpty=false)` have special purpose? There is a `ReferencesGitRepo()` outside already.

IMO the logic should work even if the repo is empty.